starter-code/server.js

```diff
@@ -71,20 +79,37 @@ app.post('/articles', function(request, response) {
 });
 
 app.put('/articles/:id', function(request, response) {
-  // TODO: Write a SQL query to update an author record. Remember that our articles now have
+  // DONE: Write a SQL query to update an author record. Remember that our articles now have
   // an author_id property, so we can reference it from the request.body.
   // TODO: Add the required values from the request as data for the SQL query to interpolate
   client.query(
-    ``,
-    []
+    `UPDATE authors
+    SET (author, authorUrl)
+    VALUES ($1, $2)
```

Comment:
This is not the syntax for an `UPDATE` statement. You need to set each one individually with `field = value`.